src/content/docs/en/guides/prefetch.mdx

```diff
@@ -51,7 +51,6 @@ Each strategy is fine-tuned to only prefetch when needed and save your users' ba
 
 - If a visitor is using [data saver mode](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/saveData) or has a [slow connection](https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/effectiveType), prefetch will fallback to the `tap` strategy.
 - Quickly hovering or scrolling over links will not prefetch them.
-- Links that use the `viewport` or `load` strategy are prefetched with a lower priority to avoid clogging up the network.
```

Comment:
This will no longer be the case as all strategies will internally use `<link rel="prefetch">` which has the same (low) priority.